docs/HISTORY.txt

```diff
@@ -5,7 +5,9 @@ Changelog
 1.5.3 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Add support for Plone 5.1.x. [mbaechtold]
+- Fit newer vers. of select2 into keywordwidget resources to prevent conflict
```

Comment:
What do we gain by abbreviating "version" with "vers."?

Reply:
👍